tutorials/python/python-type-driven-development/tutorial.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Type annotations in Python
+published: 2019-08-01
+tags: Python, typing, types, tdd
+language: python
+author-name: Carlos Villavicencio
+twitter-profile: po5i
+github-profile: po5i
+description: Learn how to improve the software quality and readability in our Python code by using the built-in typing system as it complements other software development processes like type-driven development and test-driven development.
+---
+
+_Type-Driven Development_ is a technique in which we plan our program development by adding types first and then write the code that satisfies that type. It brings many benefits such as increasing the robustness, accuracy, testability, readability, and extensibility of your code by taking advantage of the language’s type system. We strongly suggest as follow up reading [_Type-Driven Development with Idris_][tdd-idris] and extrapolate these practices to other languages.
```

Comment:
Can you add more information about the link to Edwin Brady's book or move it to the end of the tutorial? I didn't immediately noticed that this link is the one being referenced below when you cite Brady.